nixos/modules/services/misc/unpackerr.nix

```diff
@@ -0,0 +1,66 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
```

Comment:
Can you try to prevent such a wide-scoped `with`-statement?
You can import the parts of lib, that you need with something like:

```nix
let
   inherit (lib) mkEnableOption ... ;
in
```